### PR TITLE
docs(selects): Add KB for all selected items

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

**Comment by ntacheva** [comment]
Not sure why but the middle divs render a little bit below the outer ones:
image

File: components/combobox/data-bind.md [hunk 1]
```diff
@@ -62,9 +62,11 @@ You can bind the ComboBox to a model in your application. This is useful when yo
...
 To bind the ComboBox to a model:
 
-1. populate its `Data` property with the collection of items you want in the dropdown
-1. set the `TextField` and `ValueField` properties to point to the corresponding names of the model
-1. set the `Value` property to the intial value of the model. If not set, it will be populated with the first item in the data source.
+1. Populate its `Data` parameter with the collection of items you want in the dropdown.
+1. Set the `TextField` and `ValueField` parameters to point to the corresponding property names of the model.
+1. Set the `Value` property to the intial value of the component (optional).
+
+> The `TextField` and `ValueField` parameters should point to model properties, which are of **primitive** type (`int`, `string`, etc.). The `Value` and `ValueField` types should match and also be primitive.
```
Comment (marin-bratanov) [inline components/combobox/data-bind.md:69]:
```suggestion
> The `TextField` and `ValueField` parameters must point to model properties, which are of **primitive** type (`int`, `string`, etc.). The `Value` and `ValueField` types must match and also be primitive.
```

File: components/dropdownlist/data-bind.md [hunk 2]
```diff
@@ -62,9 +62,11 @@ You can bind the DropDownList to a model in your application. This is useful whe
...
 To bind the DropDownList to a model:
 
-1. populate its `Data` property with the collection of items you want in the dropdown
-1. set the `TextField` and `ValueField` properties to point to the corresponding names of the model
-1. bind the `Value` parameter to the desired field of the view-model. It is populated from the `ValueField` so it must match the its type.
+1. Populate its `Data` parameter with the collection of items you want in the dropdown.
+1. Set the `TextField` and `ValueField` parameters to point to the corresponding property names of the model.
+1. Set the `Value` property to the intial value of the component (optional).
+
+> The `TextField` and `ValueField` parameters should point to model properties, which are of **primitive** type (`int`, `string`, etc.). The `Value` and `ValueField` types should match and also be primitive.
```
Comment (marin-bratanov) [inline components/dropdownlist/data-bind.md:69]:
```suggestion
> The `TextField` and `ValueField` parameters must point to model properties, which are of **primitive** type (`int`, `string`, etc.). The `Value` and `ValueField` types must match and also be primitive.
```

File: knowledge-base/dropdown-all-items-are-selected.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    </div>
+</div>
+
+<style>
+    #dropdown-container > div {
+        display: inline-block;
+        width: 300px;
+        height: 300px;
+        padding: 1em;
+        background: #fdd;
```
Comment (dimodi) [inline knowledge-base/dropdown-all-items-are-selected.md:109]:
```suggestion
        background: #fdd;
        vertical-align: top;
```

**Comment by dimodi** [comment]
> Not sure why but the middle divs render a little bit below the outer ones:

Fixed, thanks. Didn't occur on Safari.

**Pull request merged**